src/main/webapp/assets/modules/common/validator.ser.js

```diff
@@ -6,6 +6,7 @@ angular.module("Common").factory("validatorService", function ($translate) {
       $.each(elems, function(i, elem) {
         $(elem).hasClass("ng-invalid-required") && (error[$(elem).attr("ng-model")] = $translate.instant('requiredThisField'));
         $(elem).hasClass("ng-invalid-email") && (error[$(elem).attr("ng-model")] = $translate.instant('emailInvalid'));
+        $(elem).hasClass("ng-invalid-required") && (error[$(elem).attr("ng-model")] = $translate.instant('requiredThisField'));
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Expected an assignment or function call and instead saw an expression.